src/js/App/RootApp.js

```diff
@@ -12,8 +13,30 @@ const RootApp = ({ activeApp, activeLocation, appId, config, pageAction, pageObj
   const isGlobalFilterEnabled =
     (!globalFilterHidden && activeLocation === 'insights') || Boolean(localStorage.getItem('chrome:experimental:global-filter'));
   const scalprum = useScalprum(config);
+  const remoteModule = useSelector(({ chrome }) => chrome?.activeSection?.module);
+  const insightsContentRef = useRef(null);
+  const legacyMainContainerRef = useRef(null);
+  useEffect(() => {
+    const contentElement = document.getElementById('root');
+    if (contentElement) {
+      legacyMainContainerRef.current = contentElement;
+    }
```

Comment:
Is this really needed? We are not doing anything with the `legacyMainContainerRef` later on, we might want to change line 25 `insightsContentRef.current.appendChild(legacyMainContainerRef.current);` if we want to keep it.

Reply:
I think you are right.